dataset_config/1000_genomes/ui.json

```diff
@@ -8,6 +8,7 @@
 //   listed here. ui_facet_description is optional.
 {
   "search_placeholder_text": "Try typing exome or center",
+  "enable_search": false,
```

Comment:
Let's delete. We want to make sure things work if ui.json doesn't have this flag, so we can test that out with 1000 Genomes.